index.js

```diff
@@ -102,6 +102,10 @@ module.exports.lintFiles = (patterns, opts) => {
 			});
 		}
 
+		if (paths.filter(path => path.includes('.ts')).length > 0) {
```

Comment:
You should use [path.extname](https://nodejs.org/api/path.html#path_path_extname_path) here instead of `includes`. I guess it should also includes `.tsx` files as well.